open_seq2seq/encoders/ds2_encoder.py

```diff
@@ -229,7 +202,8 @@ def _encode(self, input_dict):
       else:
         src_length = (src_length + strides[0] - 1) // strides[0]
 
-      top_layer = conv2d_bn_actv(
+      top_layer = conv_bn_actv(
+        layer = tf.layers.conv2d,
```

Comment:
Another codestyle thing: you should not have spaces around "=" sign here: "layer=tf.layers.conv2d,". Btw, you can configure your code editor to show you all these things. We generally want to be compliant with PEP8 standard with the exception that we use 2 spaces for indentation.